server/etcdserver/server_test.go

```diff
@@ -599,6 +599,11 @@ func TestApplyConfChangeError(t *testing.T) {
 func TestApplyConfChangeShouldStop(t *testing.T) {
 	cl := membership.NewCluster(zaptest.NewLogger(t))
 	cl.SetStore(v2store.New())
+
+	be, _ := betesting.NewDefaultTmpBackend(t)
```

Comment:
Could we merge this commit (at least the addition of Backends in tests) as a separate PR?

Reply:
just raised https://github.com/etcd-io/etcd/pull/19573 PTAL